inc/entity.class.php

```diff
@@ -541,6 +634,7 @@ public static function install(Migration $migration)
                     `end_date` timestamp NULL DEFAULT NULL,
                     `quantity` int NOT NULL DEFAULT '0',
                     `overconsumption_allowed` tinyint NOT NULL DEFAULT '0',
+                    `low_credit_alert` int DEFAULT NULL,
```

Comment:
The field will not be created if the table already exists, in the case of an update. You must also add it in the `else` just below.